### ISSUE TITLE
[BUG] fix notebooks top level directory persistence

### ISSUE THREAD

**Issue by webcoderz** [graphistry/graphistry-helm#43]
**Describe the bug**
Fix notebooks top level directory so files are persisted in between restarts.

**Comment by lmeyerov** [comment]
Oh is this not happening? that seems like a k8s p1

**Comment by webcoderz** [comment]
![image](https://github.com/graphistry/graphistry-helm/assets/19884161/32c87637-fe85-426d-b79b-7260831e3871)
inside all of these directories are persistent, just the top level currrent dir is not , i just have to add an extra mount point or something here

**Comment by lmeyerov** [comment]
Ping

**Comment by webcoderz** [comment]
Easy fix

**Comment by lmeyerov** [comment]
@webcoderz  can we close?

**Comment by webcoderz** [comment]
No Percy was going to work on it after OT

**Comment by webcoderz** [comment]
Since I don't need arcos for a while I'll unmount those volumes and continue testing with new volumes so no data loss on Monday

**Comment by lmeyerov** [comment]
ok tho can't spend a lot of cycles here, he needs to move on to horiz scaling

**Comment by aucahuasi** [comment]
> No Percy was going to work on it after OT

Yes, this task was part of my TODO list, but I could not allocate time yet.
Now that I finished some PRs I can take a look here.

> Easy fix

@webcoderz I remember the issue, if you have any advice on the points I need to touch, let me know please.

**Comment by lmeyerov** [comment]
let's finish otel stuff first just b/c bbai deploy deadline, tho yeah if you have idle cycles to scope this out in prep, that sounds useful!
